src/main/java/uk/gov/pay/directdebit/payments/model/GoCardlessEvent.java

```diff
@@ -12,9 +12,24 @@
     //todo action should be typed (see https://developer.gocardless.com/api-reference/#events-payment-actions and the equivalent for other resource_types
     private String action;
     private GoCardlessResourceType resourceType;
+    private String resourceId;
     private String json;
+    private String detailsCause;
```

Comment:
Has there been any discussion whether to create types for the new properties rather than using just using `String`? @alexbishop1 has done some great work to remove using `String` where it makes sense and improves readability and the best time to introduce it would be when we first add the new parameters.